mozilla/fx-private-relay
# MPP-1908 - FAQ for phone masking

This PR fixes MPP-1908
 

# New feature description 

Adds phone masking FAQ questions and answers 
 
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/3924990/193915286-e625fb5c-fd3b-47bd-a947-a9c9ac0c8e53.png">
 
# How to test

Go to the FAQ page and observe 20 new FAQ questions/answers for phone masking - all behind the `phones` flag. 

# Checklist
